Packs/Cyberwatch/Integrations/Cyberwatch/Cyberwatch.py

```diff
@@ -1189,6 +1473,158 @@ def fetch_security_issue_command(client: Client, args: Dict[str, Any]):
     )
 
 
+def list_sysadmin_assets_command(client: Client, args: Dict[str, Any]):
+    assets = client.get_sysadmin_assets(args)
+
+    if len(assets) == 0:
+        raise DemistoException("No Sysadmin assets found")
```

Comment:
it should not be an exception, it's a valid response, please send it asa demisto results not exception